plan/planbuilder.go

```diff
@@ -945,6 +946,7 @@ func (b *planBuilder) buildInsert(insert *ast.InsertStmt) Plan {
 	if b.err != nil {
 		return nil
 	}
+	insertPlan.ResolveIndices()
```

Comment:
Why remove it to here?

Reply:
We do `ResolveIndices` after building physical plan.

Reply:
I moved `ResolveIndices` to interface `PhysicalPlan`, so `Insert`/`Update`/`Delete` can call resolveIndices individually.